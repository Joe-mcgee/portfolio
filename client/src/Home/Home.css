
.Home {
  width: 100%;
  height:100%;
  display: grid;
  grid-template-columns: 1fr 1fr;
  grid-template-rows: 1fr 1fr;

}
.Portrait {
  width: 100%;
  grid-row: 2;
  grid-column: 1;
  opacity: 0.8;
}


.Header {
  grid-row: 1;
  grid-column: 1/3;

  display: grid;
  grid-template-rows: 1fr 24%;
  grid-template-columns: 38.2% 61.8%;
  background: linear-gradient(22deg, rgba(255,255,255,1), rgba(255,255,255,0));


}


/*Header CSS*/

.Header .hi {
  grid-row: 1;
  grid-column: 2;
  display: flex;
}

.Header .hi p {
align-self: flex-end;
}

.Header .title {
  grid-row: 2;
  grid-column: 2;
  display: flex;
}


.Header  .title .name .epiloge {
  font-size: 12px;
  text-align: right;

}


/*snippit*/

.About {
  grid-column: 2;
  grid-row: 2;
  background: linear-gradient(78deg, rgba(255,255,255,1), rgba(255, 255, 255, 0.38));
  display: grid;
  grid-template-rows: 1fr 1fr ;
  grid-template-columns: 1fr ;
}



.About .letter {
grid-row: 1;
grid-column: 2;
  background: linear-gradient(37.5deg, rgba(255,255,255,1), rgba(255, 255, 255, 0.38));

display: flex;
align-items: center;
text-align: center;


}

.About .business-call{
grid-row: 2;
grid-column:2;
background: linear-gradient(78deg, rgba(255,255,255,1), rgba(255, 255, 255, 0.38));
display: flex;
align-items: center;
text-align: center;

}