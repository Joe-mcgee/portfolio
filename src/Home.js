import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';


class Home extends Component {
  constructor(props) {
    super(props)
    this.state = {

    }
  }

  render () {
    return (
      <div className='Home'>
      </div>
      )
  }
}

export default Home