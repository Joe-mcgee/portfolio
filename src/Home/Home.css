.Nav {
  grid-row: 1/2;
  grid-column: 4/5
}