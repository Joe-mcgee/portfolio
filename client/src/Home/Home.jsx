import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './Home.css';



class Home extends Component {
  constructor(props) {
    super(props)
    this.state = {

    }
  }

  render () {
    return (
      <div className='Home'>
        <div className='Header'>
            <div className='hi'><p>Hi, I'm... </p></div>
          <div className='title'>
            <h1 className='name'>Taylor Hulsmans
            <p className='epiloge'>a Web Developer</p></h1>
          </div>
        </div>
        <img className='Portrait' src={require('./picture.jpg')} />
        <div className='About' >
            <p className='letter'>Pleased to resonate into your existence</p>
            <p className='business-call'>Are you in need of Web Development Resources</p>
          </div>
          <div className='interests'><p></p></div>
        </div>

      )
  }
}

Home.propTypes = {

}
export default Home