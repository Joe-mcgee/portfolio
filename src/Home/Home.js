import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './Home.css';



class Home extends Component {
  constructor(props) {
    super(props)
    this.state = {

    }
  }

  render () {
    return (
      <div className='Home'>
      <p>Taylor Hulsmans</p>
      <img />
      <p> This is My site</p>
      </div>
      )
  }
}

Home.propTypes = {

}
export default Home