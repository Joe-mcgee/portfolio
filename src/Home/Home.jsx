import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './Home.css';



class Home extends Component {
  constructor(props) {
    super(props)
    this.state = {

    }
  }

  render () {
    return (
      <div className='Home'>
        <div className='Header'>
            <div className='hi'><p>Hi, I'm... </p></div>
          <div className='title'>
            <h1 className='name'>Taylor Hulsmans</h1>
          </div>
        </div>
        <img className='Portrait' src={require('./picture.jpg')} />
        <div className='About' >
          <div className= 'snippit'>
            <p>I'm a Web Developer based from Calgary, Ab.</p>
          </div>
        </div>
      </div>
      )
  }
}

Home.propTypes = {

}
export default Home