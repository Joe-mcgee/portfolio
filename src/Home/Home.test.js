import React from 'react';
import ReactDOM from 'react-dom';

import Home from './Home';
import './Home.css';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Home />, div);
  ReactDOM.unmountComponentAtNode(div);
})

describe('Home', () => {
  it('Should be defined', () => {
    expect(Home).toBeDefined();

  });

  xit('Should render a title', () => {
    const title = renderer.create(<p>Taylor Hulsmans</p>).toJSON();
      expect(title).toMatchSnapshot();
  });

  xit('Should render picture', () => {

  });

  xit('Should render a tagline', () => {

  })

  });
