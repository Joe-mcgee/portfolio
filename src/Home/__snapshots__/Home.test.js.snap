// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Should render a tagline 1`] = `
<p
  class="snippit"
/>
`;

exports[`Home Should render a title 1`] = `
<p>
  Taylor Hulsmans
</p>
`;

exports[`Home Should render picture 1`] = `
<img
  class="Portrait"
/>
`;
